src/xarm-commander.cpp

```diff
@@ -1,40 +1,69 @@
-
+#include <easyloggingpp/src/easylogging++.h>
 #include <xarm/wrapper/xarm_api.h>
 
 #include <CLI11/include/CLI/CLI.hpp>
 #include <iostream>
 #include <string>
 
-int main(int argc, char **argv) {
-    // Default values (to avoid magic numbers) => not sure if this is the
-    // way to go (maybe using namespace, seperate headerfile?)
-    constexpr float kDefaultPosX = 300;
-    constexpr float kDefaultPosY = 0;
-    constexpr float kDefaultPosZ = 200;
-    constexpr float kDefaultPosRoll = 180;  // [Deg] by default
-    constexpr float kDefaultPosPitch = 0;   // [Deg] by default
-    constexpr float kDefaultPosYaw = 0;     // [Deg] by default
-
-    constexpr int kAllServo = 8;
+namespace constants {
+// Default values (to avoid magic numbers)
+constexpr float kDefaultPosX = 300;
+constexpr float kDefaultPosY = 0;
+constexpr float kDefaultPosZ = 200;
+constexpr float kDefaultPosRoll = 180;  // [Deg] by default
+constexpr float kDefaultPosPitch = 0;   // [Deg] by default
+constexpr float kDefaultPosYaw = 0;     // [Deg] by default
+
+constexpr int kAllServo = 8;
+
+// Logging levels
+constexpr int kLogInfo = 1;
+constexpr int kLogDebug = 2;
+constexpr int kLogEval = 3;
+}  // namespace constants
+
+INITIALIZE_EASYLOGGINGPP
+
+auto InitXarm(const std::string &port) -> XArmAPI * {
+    // TODO(jo-bru): redirect stdout to logger (using pipes?)
+    auto *arm = new XArmAPI(port);
+    return arm;
+}
```

Comment:
Assignment to this (kind of) global `arm` pointer may be redundant. Let's discuss it when we create the `server` command.